src/angularComponent/ngcOmniboxFieldDirective.js

```diff
@@ -30,7 +30,7 @@ export default function ngcOmniboxFieldDirective($document) {
           '{{::omnibox.autofocus === \'true\' || undefined}}');
       input.setAttribute('ng-disabled', 'omnibox.ngDisabled()');
 
-      tElement.html(input.outerHTML);
+      tElement[0].appendChild(input);
```

Comment:
Hmm, yea I guess I should to be safe. It doesn't take any transclusion but since there's no template some jerk could've added some stuff in there.